subprojects/packagefiles/whisper_cpp/meson.build

```diff
@@ -0,0 +1,66 @@
+project('whisper.cpp', ['cpp', 'c'], version: '1.5.2')
+
+cc = meson.get_compiler('c')
+
+math = cc.find_library('m', required: false)
+threads = dependency('threads')
+
+if build_machine.system() == 'linux'
```

Comment:
1. `host_machine`, not `build_machine`
2. checking for `cc.get_argument_syntax() == 'gcc'` would be better IMO.